firebase-firestore/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Unreleased (21.3.1)
 - [feature] Added `Query.limitToLast(n: long)`, which returns the last `n`
   documents as the result.
+- [changed] Transactions previously required that every document read in a
```

Comment:
21.3.1 is released. 

Can you move `21.3.1` to below and create `21.3.2` instead?

Reply:
Done. Thanks!